metadata-models/src/main/pegasus/com/linkedin/common/GlossaryTermAssociation.pdl

```diff
@@ -7,5 +7,10 @@ record GlossaryTermAssociation {
    /**
     * Urn of the applied glossary term
     */
+    @Searchable = {
+        "fieldName": "glossaryTerms",
+        "fieldType": "TEXT_PARTIAL",
+        "hasValuesFieldName": "hasTerms"
```

Comment:
do you expect to use the `hasTerms` filter? This filters datasets by those that have terms vs those that don't. How would you imagine this query being used?

Reply:
no we dont need hasTerms, will remove it